widget_driver/doc/widget_drivers.md

```diff
@@ -83,7 +83,7 @@ class CounterWidgetDriver extends WidgetDriver {
   }
 
   @TestDriverDefaultValue('The title of the counter')
-  String get counterTitle => _localization.counterTitle;
+  String get counterTitle => _localizationLocator<Localization>().counterTitle;
```

Comment:
As I mentioned before.. The locator is not tied to the Localization.. So maybe we name it differently..
Maybe: `contextLocator` or `locator` or `dependencyLocator`